nesta_ds_utils/nesta_io.py

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Is there a reason/style rationale to include a newline at the start of a file? Not something I usually do, but interested to know.

As a potential addition, lots of libraries include a module docstring at the top of the file that explains a bit about the contents of the utilities. I wonder if this can also be used to auto-generate text in the docs?